src/platform-includes/session-replay/setup-session-replay/javascript.mdx

```diff
@@ -0,0 +1,52 @@
+To set up the integration, add the following to your Sentry initialization. Several options are supported and passable via the integration constructor. See the [configuration sections](/platforms/javascript/session-replay/custom-instrumentation/) for more details.
+
+```javascript {tabTitle: ESM}
+import * as Sentry from "@sentry/browser";
+import { Replay } from "@sentry/replay";
```

Comment:
Here it's exactly the same as above, the `@sentry/browser` import wouldn't be correct for Framework SDKs. WDYT if (for now) we just add a comment to use the framework SDK here and in a follow-up PR later, we add platform-specific code snippets? This is how we do it for the Performance monitoring docs.
```suggestion
```javascript {tabTitle: ESM}
// import Sentry from your framework SDK (e.g. @sentry/react) instead of @sentry/browser 
import * as Sentry from "@sentry/browser";

import { Replay } from "@sentry/replay";
```